BSP/STM32F413/Src/BSP_SPI.c

```diff
@@ -87,15 +87,15 @@ static inline void SPI_WaitTx(spi_port_t port){
 /** SPI_WriteRead
  * @brief   Sends and receives a byte of data on the SPI line.
  * @param   txData single byte that will be sent to the device.
- * @return  rxData single byte that was read from the device.
+ * @return  rxData single byte that was read from the device. Returns 0xFF00 on error
```

Comment:
no magic numbers:
https://en.wikipedia.org/wiki/Magic_number_(programming)

give this return value a ```#define``` or (better) an ```enum```